### PR TITLE
Azure service operator install on preview

### PR DESCRIPTION
**Pull request by ciaranca**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-10591

### Change description ###

Kustomize files for preview and sealed secret

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: apps/azureserviceoperator-system/preview/sealed-secrets/aso-controller-settings.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
```
Comment (thomast1906) [inline apps/azureserviceoperator-system/preview/sealed-secrets/aso-controller-settings.yaml:4]: could you tidy up some values? `null` etc

File: apps/azureserviceoperator-system/preview/base/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../sealed-secrets
```
Comment (thomast1906) [inline apps/azureserviceoperator-system/preview/base/kustomization.yaml:5]: Do we need this folder `sealed-secrets` or can we just add into preview/base?

**Comment by thomast1906** [comment]
Can osba & aso run together?

File: apps/azureserviceoperator-system/preview/base/kustomization.yaml (hunk 2 above)
Comment (ciaranca) [reply apps/azureserviceoperator-system/preview/base/kustomization.yaml:5]: Yeh sure, I'll tidy this. I was using OSBA as a sort of reference.

**Pull request closed without merging**